src/meta/processors/indexMan/RebuildEdgeIndexProcessor.cpp

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/ActiveHostsMan.h"
+#include "meta/processors/admin/AdminClient.h"
+#include "meta/processors/indexMan/RebuildEdgeIndexProcessor.h"
+
+DECLARE_int32(heartbeat_interval_secs);
+
+namespace nebula {
+namespace meta {
+
+void RebuildEdgeIndexProcessor::process(const cpp2::RebuildIndexReq& req) {
```

Comment:
RebuildEdgeIndexProcessor and RebuildTagIndexProcessor looks similar.  Could we unify them?

Reply:
Because we are using `RebuildIndexReq` as parameter both in rebuild tag index and edge, how to distinguish them? Or we support RebuildTagIndexReq and RebuildEdgeIndexReq?

Reply:
You could extract the common logic into a base class such as RebuildIndexProcessor

Reply:
I think we should keep it cleaning and I have move the similar code into `RebuildIndexProcessor` and the admin client calling in `adminCaller`

Reply:
`adminCaller` maybe not a good name, do you have better one ?